custom_reports/object_management_api/definitions/product_custom_dimension_details.yaml

```diff
@@ -0,0 +1,14 @@
+type: object
+allOf:
+  - $ref: '../index.yaml#/components/schemas/ProductCustomDimension'
```

Comment:
oh my, our PublicDocs are horrible to read, with all this jumping between files :/